pkg/mappings.bzl

```diff
@@ -311,6 +311,15 @@ def _pkg_files_impl(ctx):
                     else:
                         src_dest_paths_map[rf] = dest_path
 
+                if (
```

Comment:
This is a bit of magic, and it is repeated below.
Can you extract it out to a routine (maybe in pkg/private/util.bzl) with a good name (maybe "repo_mapping_manifest_exists"?)

Also, there is a comment about bzlmod enabled. That should go with the routine to clarify why this check is the way it is.

Reply:
I've split the logic into `get_repo_mapping_manifest`, please take a look. 🙏 

RE: the comment about bzlmod they are not really related to the check, but the fact that we'd need to include repo_mappings in the runfiles directory. So I am keeping them with where we add repo mappings into the mapping_context